### PR TITLE
[hi] Localize content/en/docs/reference/glossary/cni.md

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com> 
Fixes #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/cni.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+aka: 
+tags:
+- networking 
+---
+ कंटेनर नेटवर्क इंटरफ़ेस (CNI) प्लगइन एक प्रकार का नेटवर्क प्लगइन है जो appc/CNI विनिर्देश का पालन करता है।
+<!--more--> 
+
+* कुबेरनेट्स और CNI के बारे में जानकारी के लिए, देखें [यह](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni).
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cni.md:17]: This is the same line as below.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Duplicate lines.

**Review by amolmote — COMMENTED** [review]
please check changes again

**Review by amolmote — COMMENTED** [review]
please check changes again and merge it

File: content/hi/docs/reference/glossary/cni.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: कंटेनर नेटवर्क इंटरफ़ेस (CNI)
+id: cni
+date: 2018-05-25
+full_link: /docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni
+short_description: >
+    कंटेनर नेटवर्क इंटरफेस (CNI) प्लगइन्स एक प्रकार का नेटवर्क प्लगइन है जो appc/CNI विनिर्देश का पालन करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cni.md:7]:
```suggestion
    कंटेनर नेटवर्क इंटरफेस (CNI) प्लगइन्स एक प्रकार का नेटवर्क प्लगइन हैं जो appc/CNI विनिर्देश का पालन करते हैं।
```

File: content/hi/docs/reference/glossary/cni.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+aka: 
+tags:
+- networking 
+---
+ कंटेनर नेटवर्क इंटरफ़ेस (CNI) प्लगइन एक प्रकार का नेटवर्क प्लगइन है जो appc/CNI विनिर्देश का पालन करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cni.md:14]:
```suggestion
 कंटेनर नेटवर्क इंटरफ़ेस (CNI) प्लगइन एक प्रकार का नेटवर्क प्लगइन हैं जो appc/CNI विनिर्देश का पालन करते हैं।
```

File: content/hi/docs/reference/glossary/cni.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+<!--more--> 
+
+* कुबेरनेट्स और CNI के बारे में जानकारी के लिए [यह](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni) देखें।
+* कुबेरनेट्स और CNI के बारे में जानकारी के लिए, देखें ["नेटवर्क प्लगइन्स"](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni)।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cni.md:18]:
```suggestion
* कुबेरनेट्स और CNI के बारे में जानकारी के लिए, ["नेटवर्क प्लगइन्स"](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#cni) देखें।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by amolmote** [comment]
thank you @anubha-v-ardhan , I will  rebase all the commits.

**Review by anubha-v-ardhan — APPROVED** [review]
/lgtm
/assign
/approve

Thank you @amolmote :)

**Pull request merged**